src/component/Header.js

```diff
@@ -1,6 +1,7 @@
 
 
```

Comment:
There are quite a bit of random newlines around. Since this is an example, could we clean these up?

Reply:
Not random, I like whitespace. I'm weird that way. I'll remove 'em.

Reply:
If there's a reason then whatever. I just want the example to be clean. =]